sdk/metricsadvisor/perf-tests/ai-metrics-advisor/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@azure-tests/perf-ai-metrics-advisor",
+  "version": "1.0.0",
+  "description": "",
+  "main": "",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
```

Comment:
- please have a `dependencies` section for things that are imported by the tests and keep `devDependencies` for things needed to run the tests. See text analytics perf tests for an example.
- add `prettier` as a dev dependency and make sure the `format` script works.
- add `eslint` as a dev dependency and make sure the lint commands are defined as follows:
```
    "lint:fix": "eslint --no-eslintrc -c ../../../.eslintrc.internal.json package.json test --ext .ts --fix --fix-type [problem,suggestion]",
    "lint": "eslint --no-eslintrc -c ../../../.eslintrc.internal.json package.json test --ext .ts",
```
and make sure `rushx lint` passes.

Reply:
lint command is failing - S C:\code\sdk-for-js\sdk\metricsadvisor\perf-tests\ai-metrics-advisor> npm run lint

> @azure-tests/perf-ai-metrics-advisor@1.0.0 lint C:\code\sdk-for-js\sdk\metricsadvisor\perf-tests\ai-metrics-advisor
> eslint package.json src test --ext .ts -f html -o metrics-advisor-perf-test-lintReport.html || exit 0


Oops! Something went wrong! :(

ESLint: 7.22.0

ESLint couldn't find the config "plugin:@azure/azure-sdk/azure-sdk-base" to extend from. Please check that the name of the config is correct.

The config "plugin:@azure/azure-sdk/azure-sdk-base" was referenced from the config file in "C:\code\sdk-for-js\sdk\.eslintrc.json".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.

Reply:
Because you did not update the linting commands as I mentioned in my previous comment.

Reply:
Please remove `"@azure/eslint-plugin-azure-sdk": "^3.0.0"` from your dependencies and do not forget to split `devDependencies` into `dependencies` and `devDependencies` as I explained before. See [text analytics](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/textanalytics/perf-tests/text-analytics/package.json) for an example.